README.md

```diff
@@ -1,350 +1,15 @@
-<<<<<<< HEAD
-=======
 HCAquery
 ================
->>>>>>> 765bac970600c72ce892c6b4a3d08c457afb1811
 
 Load Metadata
 
-``` r
-library(HCAquery)
```

Comment:
Shouldn't we still show how to load the relevant libraries? We could just set `echo=TRUE` but keep the others at `FALSE`, maybe?

Reply:
done